<launch>

    <arg name="markerSize" default="0.08"/>    <!-- in m -->
    <arg name="ref_frame" default="camera_rgb_optical_frame"/>

    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher">
        <remap from="/camera_info" to="/camera/rgb/camera_info" />
        <remap from="/image" to="/camera/rgb/image_raw" />
        <param name="image_is_rectified" value="False"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>
        <param name="camera_frame"       value="$(arg ref_frame)"/>
    </node>

</launch>

